tests/ui/unseparated_prefix_literals.rs

```diff
@@ -17,4 +21,8 @@ fn main() {
     let _okf2 = 1_f32;
     let _failf1 = 1.5f32;
     let _failf2 = 1f32;
+
+    // Counter example
+    let _ = line!();
+    let _ = lit_from_macro!();
```

Comment:
It's ok, that internal macros get linted, as long as the suggestion is correct, which is the case here.